src/test/java/org/apache/drillui/test/framework/testng/unsecure/rest/BaseRestTest.java

```diff
@@ -23,7 +23,13 @@
 public class BaseRestTest {
   @BeforeSuite
   public void setupRestAssured() {
-    RestAssured.port = Integer.parseInt(TestProperties.get("DRILL_PORT"));
```

Comment:
@Atri0 Could you please revert the changes to this file? We should have host and port data in the init.properties file.